src/generic/generic-component-site-header/generic-component-site-header.twig

```diff
@@ -68,7 +68,7 @@
 
   <div class="ecl-container ecl-site-header__banner">
     {% include '@ecl/generic-component-logo' with {
-      'to': 'https://ec.europa.eu',
+      'href': 'https://ec.europa.eu',
       'title': 'Home - European Commission',
```

Comment:
Could you please make this configurable from the component's API? (In the same way you've done [here](https://github.com/ec-europa/europa-component-library/pull/660/files#diff-ee408b90955ce3feb819d7fb7440e89bL5))

It's a question for the whole file, as there's also the `'heading': 'Select your language'`

Reply:
I've not updated component-site-header yet :) just the context here

Reply:
:+1: